### PR TITLE
Resolved #38

### LINKED ISSUE #38: Merchant Dashboard Statistics - Favorite Customers
As a merchant,
When I visit my merchant dashboard
Then I see the names of the top 5 customers
who have conducted the largest number of successful transactions with my merchant
And next to each customer name I see the number of successful transactions they have
conducted with my merchant

### PR DESCRIPTION
**Pull request by WadeNaughton**
Merchant Dashboard statistics story with passing tests. Model and features.

### CODE REVIEW

**Review by echon006 — COMMENTED** [review]
Created AR together on zoom - Story and test passing

**Review by arnaldoaparicio — COMMENTED** [review]
Coolio

File: Gemfile [hunk 1]
```diff
@@ -43,8 +43,9 @@ group :development, :test do
   gem 'capybara'
   gem 'launchy'
   gem 'simplecov'
-  gem 'shoulda-matchers' 
+  gem 'shoulda-matchers'
   gem 'factory_bot_rails'
+  gem 'orderly'
```
Comment (tjroeder) [inline Gemfile:48]: Nice, I forgot about adding this one at the beginning.

File: spec/models/merchant_spec.rb [hunk 2]
```diff
@@ -1,19 +1,79 @@
 require 'rails_helper'
 
 RSpec.describe Merchant, type: :model do
-  before do
 
-  end
```
Comment (tjroeder) [inline spec/models/merchant_spec.rb:6]: Yeah, don't know why this was here.

**Review by tjroeder — APPROVED** [review]
Changes look good.

**Pull request merged**